cmd/run-once-duration-override/runoncedurationoverride.go

```diff
@@ -11,14 +11,21 @@ import (
 
 	admissionv1 "k8s.io/api/admission/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	apiserver "k8s.io/apiserver/pkg/server"
+	"k8s.io/apiserver/pkg/server/mux"
+	apiserveroptions "k8s.io/apiserver/pkg/server/options"
 	restclient "k8s.io/client-go/rest"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+const (
+	DefaultRODOMetricsPort = 10258
```

Comment:
It might be tricky to use the right port. https://kubernetes.io/docs/reference/networking/ports-and-protocols/ mentions only the control plane ports. 10258 is used by the descheduler. I would prefer moving away from 102 prefixed port number. To make sure we have "more" OpenShift port numbers.

Reply:
We already use 9448 for the webhook endpoint. Let's use the same port for the metrics as well.